gocd/checkers.suse.gocd.yaml.erb

```diff
@@ -131,7 +131,6 @@ pipelines:
 
                osc -A https://api.suse.de staging -p SUSE:SLE-15-SP4:GA rebuild
                osc -A https://api.suse.de staging -p SUSE:SLE-15-SP4:GA list --supersede
-               osc -A https://api.suse.de staging -p SUSE:SLE-15-SP4:GA adi --by-develproject
```

Comment:
no need to run 'adi' at all? Even without `--by-develproject`?

Reply:
SP4 and MicroOS don't have rings ...